apps/file_manager.talon

```diff
@@ -14,20 +14,22 @@ go user: user.file_manager_open_directory(path.user_home())
 go back: user.file_manager_go_back()
 go forward: user.file_manager_go_forward()
 (go parent | daddy): user.file_manager_open_parent()
-^follow <number>$:
-    directory = user.file_manager_get_directory_by_index(number - 1)
+^follow numb <number_small>$: 
```

Comment:
Something seems wrong here - why is this PR to add create_spoken_forms changing the file_manager commands?

Reply:
With the new create_spoken_forms, the old commands conflict / don’t work. If there’s a strong preference, I don’t mind moving the file manager changes to another PR